napari/_app_model/actions/_view.py

```diff
@@ -0,0 +1,75 @@
+from napari._app_model.actions._toggle_action import ViewerModelToggleAction
+from napari._app_model.constants import MenuId
+from napari.utils.translations import trans
+
+toggle_action_details = [
+    (
+        'napari.window.view.toggle_viewer_axes',
+        trans._('Axes Visible'),
+        'axes',
+        'visible',
+    ),
+    (
+        'napari.window.view.toggle_viewer_axes_colored',
+        trans._('Axes Colored'),
+        'axes',
+        'colored',
+    ),
+    (
+        'napari.window.view.toggle_viewer_axes_labels',
+        trans._('Axes Labels'),
+        'axes',
+        'labels',
+    ),
+    (
+        'napari.window.view.toggle_viewer_axesdashed',
```

Comment:
```suggestion
        'napari.window.view.toggle_viewer_axes_dashed',
```
?

Reply:
I see that this comes from an earlier implementation, but I think it's worth updating the ID here.